bot/constants.py

```diff
@@ -0,0 +1,7 @@
+#COMANDS
```

Comment:
There must be a single space between # and the beginning of the comment, see https://www.python.org/dev/peps/pep-0008/#inline-comments